woocommerce/class-sv-wc-plugin-compatibility.php

```diff
@@ -326,6 +326,33 @@ public static function is_enhanced_admin_available() {
 	}
 
 
+	/**
+	 * Determines if the current request is for a WC REST API endpoint.
+	 *
+	 * @see \WooCommerce::is_rest_api_request()
+	 *
+	 * @since x.y.z-dev.1
+	 *
+	 * @return bool
+	 */
+	public static function is_rest_api_request() {
+
+		if ( function_exists( 'WC' ) && is_callable( [ WC(), 'is_rest_api_request' ] ) ) {
```

Comment:
Since we are already using in the second part of the condition I figured we could make the first part as strict:

```suggestion
		if ( is_callable( 'WC' ) && is_callable( [ WC(), 'is_rest_api_request' ] ) ) {
```